packages/docusaurus-types/src/i18n.d.ts

```diff
@@ -11,8 +11,10 @@ export type I18nLocaleConfig = {
   /** The label displayed for this locale in the locales dropdown. */
   label: string;
   /**
-   * BCP 47 language tag to use in `<html lang="...">` and in
-   * `<link ... hreflang="...">`
+   * BCP 47 language tag to use in:
+   * - `<html lang="...">`
+   * - `<li lang="...">`
```

Comment:
`<li>` or `<a>`, or doesn't really matter? Also need to put this into the actual docs since they are in sync.

Reply:
Replaced by 

```
`<html lang="...">` (or any other DOM tag name)
```

Somehow HTML, li, a are all the case case 🤪